packages/cli/src/cli/config/config.update.ts

```diff
@@ -62,6 +62,8 @@ export class Updater<S extends BaseConfig = BaseConfig> {
   async reconcile(): Promise<boolean> {
     const db = this.getDB();
 
+    if (this.config.id !== 'ts_topo50-gridless-2020-4.23m') return false;
```

Comment:
We never update this layer?

Reply:
this was my testing layer, it had a bunch of "undefined" keys added to it.